infrastructure/scripts/airy-conf.sh

```diff
@@ -48,15 +50,20 @@ sed -i "s/<fb_app_secret>/${config[FB_APP_SECRET]}/" ~/airy-core/api-admin.yaml
 
 sed -i "s/<fb_webhook_secret>/${config[FB_WEBHOOK_SECRET]}/" ~/airy-core/sources-facebook-webhook.yaml
 
+sed -i "s/<google_partner_key>/${config[GOOGLE_PARTNER_KEY]}/" ~/airy-core/sources-google-webhook.yaml
+sed -i "s/<google_sa_file>/${config[GOOGLE_SA_FILE]}/" ~/airy-core/sources-google-sender.yaml
+
 sed -i "s/<mail_url>/${config[MAIL_URL]}/" ~/airy-core/api-auth.yaml
 sed -i "s/<mail_port>/${config[MAIL_PORT]}/" ~/airy-core/api-auth.yaml
 sed -i "s/<mail_username>/${config[MAIL_USERNAME]}/" ~/airy-core/api-auth.yaml
 sed -i "s/<mail_password>/${config[MAIL_PASSWORD]}/" ~/airy-core/api-auth.yaml
 sed -i "s/<mail_from>/${config[MAIL_FROM]}/" ~/airy-core/api-auth.yaml
 
 # Generate random string for the ngrok webhook
-RANDOM_INGRESS_ID=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 16; echo`
-sed -i "s/<ngrok_client_string>/fb-${RANDOM_INGRESS_ID}/" ~/airy-core/sources-facebook-webhook.yaml
+GOOGLE_INGRESS_ID=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 16; echo`
+FACEBOOK_INGRESS_ID=`cat /dev/urandom | env LC_CTYPE=C tr -dc a-z0-9 | head -c 16; echo`
+sed -i "s/<ngrok_facebook_client>/fb-${FACEBOOK_INGRESS_ID}/" ~/airy-core/public-urls-config.yaml
+sed -i "s/<ngrok_google_client/gl-${GOOGLE_INGRESS_ID}/" ~/airy-core/public-urls-config.yaml
```

Comment:
Wanted to make this a one-liner, but sedly I was unable

Reply:
oh wow..

Reply:
would probably keep related lines together (like we do fb first and then google or the other way around)